package.json

```diff
@@ -26,8 +26,11 @@
   },
   "dependencies": {
     "@sphereon/ssi-types": "0.8.1-next.31",
+    "@types/jwt-encode": "^1.0.0",
     "bs58": "^5.0.0",
     "cross-fetch": "^3.1.5",
+    "jwt-decode": "^3.1.2",
```

Comment:
Hmzz, is there a need we now depend on 2 additional JWT libs, whilst we are working with a callback to create the eventual JWTs anyway?

Reply:
removed this and used jose